package.json

```diff
@@ -52,7 +52,7 @@
     "typescript": "^5.6.2"
   },
   "optionalDependencies": {
-    "@hubspot/cms-dev-server": "^0.18.10"
+    "@hubspot/cms-dev-server": "^0.18.9"
```

Comment:
This is still downgrading the dep for some reason.  Since it has that `^` it is probably fine, but still seems unintentional.